test/dummy/.env.test

```diff
@@ -1 +1 @@
-DATABASE_URL=postgres://localhost/elastic_record_test
+DATABASE_URL=postgres://localhost/elastic_record_subtest
```

Comment:
I'm not sure if this is required but this was the name of the database generated by `bundle exec rake app:db:setup`.  I wasn't able to get the tests to run until I changed this env config.

Reply:
What version of rails is in your `Gemfile.lock`?

Reply:
Rails 5.1.4